airflow/auth/managers/base_auth_manager.py

```diff
@@ -385,6 +386,10 @@ def get_fastapi_app(self) -> FastAPI | None:
         """
         return None
 
+    def get_additional_menu_items(self) -> list[MenuItem]:
```

Comment:
I kinda think `get_menu_items` is a better name.

Reply:
Agree